apis/api-journeys/src/app/modules/event/journey/journey.resolver.ts

```
@@ -3,6 +3,7 @@
 import { UseGuards } from '@nestjs/common'
 import { Args, Mutation, Parent, ResolveField, Resolver } from '@nestjs/graphql'
 import { GraphQLError } from 'graphql'
+import { omit } from 'lodash'
```

ESLint

[error] 'lodash' import is restricted from being used.

(no-restricted-imports)